Makefile

```
@@ -146,6 +159,19 @@ update-debug-minimal:
 	docker build $(CONTAINER_BUILD_PARAMS) --build-arg SERVICE=$(SERVICE) --build-arg DEBUG_PORT=$(DEBUG_PORT) -f build/debug-image/Dockerfile.assisted-service-debug build/debug-image -t $(SERVICE)
 	rm -r build/debug-image
 
+update-image: $(UPDATE_IMAGE)
+
+_update-private-registry-image: update-image
```

No need for a new target if it's an internal one and only calls to another target
```suggestion
_update-private-registry-image: $(UPDATE_IMAGE)
```

---

This target is used by the programmer to update the image (like a public api)